supervision/tracker/byte_tracker/byte_tracker.py

```diff
@@ -0,0 +1,483 @@
+        return ret
+
+    @staticmethod
+    # @jit(nopython=True)
+    def tlwh_to_xyah(tlwh):
+        """Convert bounding box to format `(center x, center y, aspect ratio,
+        height)`, where the aspect ratio is `width / height`.
+        """
+        ret = np.asarray(tlwh).copy()
+        ret[:2] += ret[2:] / 2
+        ret[2] /= ret[3]
+        return ret
+
+    def to_xyah(self):
+        return self.tlwh_to_xyah(self.tlwh)
+
+    @staticmethod
+    # @jit(nopython=True)
+    def tlbr_to_tlwh(tlbr):
+        ret = np.asarray(tlbr).copy()
+        ret[2:] -= ret[:2]
+        return ret
+
+    @staticmethod
+    # @jit(nopython=True)
+    def tlwh_to_tlbr(tlwh):
+        ret = np.asarray(tlwh).copy()
+        ret[2:] += ret[:2]
+        return ret
+
+    def __repr__(self):
+        return "OT_{}_({}-{})".format(self.track_id, self.start_frame, self.end_frame)
+
+
+# converts Detections into format that can be consumed by match_detections_with_tracks function
+def detections2boxes(detections: Detections) -> np.ndarray:
+    """
+    Convert Detections into a format that can be consumed by the match_detections_with_tracks function.
+
+    Parameters:
+        detections (Detections): An object representing the detected bounding boxes.
+
+    Returns:
+        np.ndarray: An array containing the bounding boxes' coordinates (xyxy) and their corresponding confidences.
+    """
+    return np.hstack((detections.xyxy, detections.confidence[:, np.newaxis]))
+
+
+# converts List[strack] into format that can be consumed by match_detections_with_tracks function
+def tracks2boxes(tracks: List[strack]) -> np.ndarray:
+    return np.array([track.tlbr for track in tracks], dtype=float)
+
+
+# matches our bounding boxes with predictions
+def match_detections_with_tracks(
+    detections: Detections, tracks: List[strack]
+) -> Detections:
+    """
+    Match bounding boxes from detections with existing strack objects.
+
+    Parameters:
+        detections (Detections): An object representing the detected bounding boxes.
+        tracks (List[strack]): A list of strack objects to match with the detections.
+
+    Returns:
+        Detections: An object representing the matched tracker IDs for the detections.
+    """
+    if not np.any(detections.xyxy) or len(tracks) == 0:
+        return np.empty((0,))
+
+    tracks_boxes = tracks2boxes(tracks=tracks)
+    iou = box_iou_batch(tracks_boxes, detections.xyxy)
+    track2detection = np.argmax(iou, axis=1)
+
+    tracker_ids = [None] * len(detections)
+
+    for tracker_index, detection_index in enumerate(track2detection):
+        if iou[tracker_index, detection_index] != 0:
+            tracker_ids[detection_index] = tracks[tracker_index].track_id
+
+    return tracker_ids
+
+
+class ByteTrack:
+    def __init__(
+        self,
+        track_thresh=0.25,
+        track_buffer=30,
+        match_thresh=0.8,
+        aspect_ratio_thresh=3.0,
+        min_box_area=1.0,
+        mot20=False,
```

Comment:
@dbroboflow Can you remove unnecessary parameter like `mot20`, it does not require here.